migration/testbuilder.go

```diff
@@ -10,12 +10,22 @@ import (
 	"github.com/cloudquery/cq-provider-sdk/migration/migrator"
 	"github.com/cloudquery/cq-provider-sdk/provider"
 	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+	"github.com/georgysavva/scany/pgxscan"
 	"github.com/golang-migrate/migrate/v4"
 	"github.com/hashicorp/go-hclog"
+	"github.com/jackc/pgx/v4/pgxpool"
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/assert"
 )
 
+const queryPrimaryKeys = `SELECT kcu.table_name, c.constraint_name, ARRAY_AGG(kcu.column_name::text ORDER BY kcu.ordinal_position) AS pk_cols
+FROM information_schema.table_constraints c
```

Comment:
Can you add a bit of docs on what this query is doing ? :)

Reply:
"documented"